tic-tac-toe-app/components/Game.jsx

```diff
@@ -0,0 +1,145 @@
+function Board() {
+    const [squares, setSquares] = React.useState(new Array(9).fill(null));
+
+    squares[index] = props.turn;
+    setSquares(squares[index]);
+
+    const declareWinner = () => {
+        const conditions = [
+            [0, 1, 2],
+            [3, 4, 5],
+            [6, 7, 8],
+            [0, 3, 6],
+            [1, 4, 7],
+            [2, 5, 8],
+            [0, 4, 8],
+            [2, 4, 6]
+        ];
+
+        for (let i = 0; i < conditions.length; i++) {
+            const [first, second, third] = conditions[i];
+            if (squares[first] && squares[second] === squares[first] && squares[third] === squares[first]) {
+                return squares[first];
+            }
+        }
+    }
+
+    const winner = declareWinner(squares);
+    
+    let status;
+    if (winner) {
+        status = `The winner is ${winner}`;
+    }
+
+    function clickHandle(index) {
+        if (!winner || !squares[index]) {
+            squares[index] = props.turn;
+            setSquares(squares)
+            props.setTurn(() => {
+                if (props.turn === 'X') {
+                    props.turn = 'O'
+                }
+                props.turn = 'X'
+            })
+        }
+    }
+
+    return (
+            <section className='board'>
+                <h3>The winner is {winner}</h3>
+                <section className="row row1">
+                    <Square onClick={() => pickSquare(0)} value={setSquares[0]}/>
+                    <Square onClick={() => pickSquare(1)} value={setSquares[1]}/>
+                    <Square onClick={() => pickSquare(2)} value={setSquares[2]}/>
```

Comment:
The `squares` are established on use state. Mapping through those `squares` and rendering an instance of the `<Square/>` component will make this portion of the code more efficient.